swesmith/profiles/javascript.py

```diff
@@ -315,6 +313,192 @@ def log_parser(self, log: str) -> dict[str, str]:
         return parse_log_jest(log)
 
 
+@dataclass
+class Wretch661865a6(JavaScriptProfile):
+    owner: str = "elbywan"
+    repo: str = "wretch"
+    commit: str = "661865a6642f6be26e742a90a3e0a9b9bd5542ff"
+    test_cmd: str = "npm run test -- --verbose"
+
+    @property
+    def dockerfile(self):
+        return f"""FROM node:22-bullseye
+RUN apt update && apt install -y git
+RUN git clone https://github.com/{self.mirror_name} /testbed
+WORKDIR /testbed
+RUN npm install
+RUN npm run build
+"""
+
+    def log_parser(self, log: str) -> dict[str, str]:
+        return parse_log_jest(log)
+
+
+@dataclass
+class Html5Boilerplateac08a17c(JavaScriptProfile):
+    owner: str = "h5bp"
+    repo: str = "html5-boilerplate"
+    commit: str = "ac08a17cb60a975336664c0090657a3e593f686e"
+    test_cmd: str = "npm run test -- --verbose"
+
+    @property
+    def dockerfile(self):
+        return f"""FROM node:22-bullseye
+RUN apt update && apt install -y git
+RUN git clone https://github.com/{self.mirror_name} /testbed
+WORKDIR /testbed
+RUN npm ci
+"""
+
+    def log_parser(self, log: str) -> dict[str, str]:
+        return parse_log_mocha(log)
+
+
+@dataclass
+class HighlightJS5697ae51(JavaScriptProfile):
```

Comment:
As discussed in slack perhaps we don't add `highlightjs/highlight.js` as it is in SWE-bench_Multimodal.